src/navBar/navbar.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title> Quality Education </title>
+    <link rel="stylesheet" href="NavBar_style.css">

```

Comment:
the style sheet to link with our folders so `<link rel="stylesheet" href="../template.css">`